mrmustard/math/lattice/strategies/vanilla.py

```diff
@@ -188,3 +196,134 @@ def vanilla_vjp(G, c, dLdG) -> tuple[ComplexMatrix, ComplexVector, complex]:  #
     dLdc = np.sum(G_lin.reshape(shape) * dLdG) / c
 
     return dLdA, dLdb, dLdc
+
+
+# @njit
```

Comment:
is it not jit-friendly yet? will we make it so later?

Reply:
oops nope, I just forgot to uncomment!